common/src/main/java/org/apache/uniffle/common/ReconfigurableConfManager.java

```diff
@@ -0,0 +1,53 @@
+package org.apache.uniffle.common;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+import org.apache.uniffle.common.config.ConfigOption;
+import org.apache.uniffle.common.config.RssBaseConf;
+import org.apache.uniffle.common.config.RssConf;
+import org.apache.uniffle.common.util.ThreadUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ReconfigurableConfManager {
+    private static final Logger LOGGER = LoggerFactory.getLogger(ReconfigurableConfManager.class);
+
+    private static RssBaseConf rssConf;
+    private ScheduledExecutorService scheduledThreadPoolExecutor;
+
+    private ReconfigurableConfManager(Supplier<RssBaseConf> confSupplier) {
+        rssConf = confSupplier.get();
+        scheduledThreadPoolExecutor = ThreadUtils.getDaemonSingleThreadScheduledExecutor("Refresh-rss-conf");
+        scheduledThreadPoolExecutor.schedule(() -> {
+            try {
+                rssConf.addAll(confSupplier.get());
```

Comment:
Maybe you should log more information to display the change of config options.

Reply:
Yes. This could be logged of changed for better user experience.

Reply:
Done